lib/twilio-ruby/rest/client.rb

```diff
@@ -26,6 +29,7 @@ def initialize(username=nil, password=nil, account_sid=nil, region=nil, http_cli
         @auth_token = @password
         @auth = [@username, @password]
         @http_client = http_client || Twilio.http_client || Twilio::HTTP::Client.new
+        @log_level = log_level || ENV['TWILIO_LOG_LEVEL'] || Twilio.log_level
```

Comment:
I know it's different from the implementations in other languages, but I like the approach of the caller providing a `logger` instance and we just debug log stuff if they gave us an actual instance. Then they control the level of what actually gets logged and where it gets logged. No need for env vars.